packages/devtools/lib/src/ui/drag_scroll.dart

```diff
@@ -79,45 +89,47 @@ class DragScroll {
       final Touch touch = t.touches.first;
 
       touched = true;
-      wasDraggedByTouch = false;
+      wasDragged = false;
 
       lastX = touch.client.x;
       lastY = touch.client.y;
 
       t.preventDefault();
-    });
 
-    window.onTouchEnd.listen((event) {
-      final TouchEvent t = event;
-      if (t.touches.isEmpty) {
-        touched = false;
-      }
-    });
-
-    window.onTouchMove.listen((event) {
-      final TouchEvent t = event;
-      // If there are multiple touches, always use the first.
-      final Touch touch = t.touches.first;
+      _touchMoveListener = window.onTouchMove.listen((event) {
+        final TouchEvent t = event;
+        // If there are multiple touches, always use the first.
+        final Touch touch = t.touches.first;
 
-      if (touched) {
-        final num newX = touch.client.x;
-        final num newY = touch.client.y;
+        if (touched) {
```

Comment:
can you remove the touched variable now that you stop listening as soon as the touch action ends?

Reply:
done 👍